bindings/python/test/trajectory/test_segment.py

```
@@ -235,6 +235,15 @@ def test_compute_delta_mass(
     ):
         assert segment_solution.compute_delta_mass() is not None
 
+    def test_get_thruster_dynamics(
+        self,
+        segment_solution: Segment.Solution,
+        dynamics: list,
```

Ruff (0.14.3)

Unused method argument: `dynamics`

(ARG002)